src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -743,16 +743,6 @@ object ScalafixPlugin extends AutoPlugin {
             options.ignoredScalacOptions() ++
               scalacOptionsToRelax.map(_.pattern())
           )
-      },
-      manipulateBytecode := {
-        val analysis = manipulateBytecode.value
-        if (!scalafixInvoked.value) analysis
-        else {
-          // prevent storage of the analysis with relaxed scalacOptions - despite not depending explicitly on compile,
-          // it is being triggered for parent configs/projects through evaluation of dependencyClasspath (TrackAlways)
-          // in the scope where scalafix is invoked
-          analysis.withHasModified(false)
```

Comment:
it turns that this was unnecessary, since there is scenario were a relaxed invocation would override a prior regular one and thus invalidate the cache